### PR TITLE
refactor(prejoin) use jss instead of sass in DialInDialog

### PR DESCRIPTION
**Pull request by c0m1t**
Convert Sass to JSS in `DialInDialog` component.

### CODE REVIEW

File: react/features/prejoin/components/dialogs/DialInDialog.js [hunk 1]
```diff
@@ -46,6 +48,53 @@ type Props = {
     t: Function,
 };
 
+const useStyles = makeStyles(theme => {
+    return {
+        root: {
```
Comment (robertpin) [inline react/features/prejoin/components/dialogs/DialInDialog.js:53]: Can you rename it to `dialog`? I think `root` is kinda vague

File: react/features/prejoin/components/dialogs/DialInDialog.js [hunk 2]
```diff
@@ -63,12 +112,13 @@ function DialinDialog(props: Props) {
...
         t
     } = props;
+    const classes = useStyles();
     const flagClassName = `prejoin-dialog-flag iti-flag ${getCountryCodeFromPhone(
         number
     )}`;
 
     return (
-        <div className = 'prejoin-dialog-dialin'>
+        <div className = { clsx('prejoin-dialog-dialin', classes.root) }>
```
Comment (robertpin) [inline react/features/prejoin/components/dialogs/DialInDialog.js:121]: Do we still need the `prejoin-dialog-dialin` class here? Is it used by tests? If not let's remove it please

Comment (robertpin) [inline react/features/prejoin/components/dialogs/DialInDialog.js:121]: Do we still need the `prejoin-dialog-dialin` class here? Is it used by tests? If not let's remove it please

**Review by robertpin — CHANGES_REQUESTED** [review]

**Review by robertpin — APPROVED** [review]

**Comment by robertpin** [comment]
@c0m1t please rebase this

**Comment by c0m1t** [comment]
@robertpin Done.

**Review by robertpin — APPROVED** [review]

**Pull request merged**